scripts/accordian.js

```diff
@@ -0,0 +1,76 @@
+    class AccordianComponent {
+
+    NUMBER_OF_ITEMS = 4;
+    container = document.createElement('div');
+    nodeList = [];
+
+    init = () => {
+        this.container.setAttribute('id', 'container');
+        document.body.appendChild(this.container);
```

Comment:
you are referencing the class instance name from within the class instance itself since the name of the instance is not available within the class instance itself.
typically within an es6 class you would use this.createDom()